test/release/examples/primers/forallLoops.chpl

```diff
@@ -250,13 +252,17 @@ writeln();
     single: forall; with
 .. _primers-forallLoops-with:
 
-The task intents ``in``, ``const in``, ``ref``, ``const ref``,
-and ``reduce`` can be specified explicitly using a ``with`` clause.
+The task intents ``in``, ``const in``, ``ref``, and ``const ref``
+can be specified explicitly using a ``with`` clause.
 
 An ``in`` or ``const in`` intent creates a copy of the outer variable
-for each task. A ``ref`` or ``const ref`` makes the
-shadow variable an alias for the outer variable. Updates to a ``ref``
-shadow variable are reflected in the corresponding outer variable.
+for each task. Such a shadow variable is never accessed concurrently
```

Comment:
"the underlying outer variable will not be accessed concurrently by multiple tasks". Or something to that effect, I think we're trying to highlight the difference between `in` and `ref`?